spectaql/config.yml

```diff
@@ -391,21 +391,21 @@ info:
   #   name: Commerce Help Center
   #   url: https://support.magento.com/hc/en-us
   # license:
-  #   name: Apache 2.0
-  #   url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+  #     name: Apache 2.0
+  #     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
 
   # A non-standard array of items to display in your Introduction Area
   x-introItems:
     # Can be a Title (for the Nav panel) + URL to simply add a link to somewhere
     - title: WebAPI guide
       url: https://developer.adobe.com/commerce/webapi/
     # Can be a Title (for the Nav panel) + description (for the Content panel)
-    - title: Contributing
-      description: This page is auto-generated from the GraphQL schema obtained from the source code.
-        For any issues and updates, contribute to the corresponding module directly.
+    # - title: Contributing
+    #   description: This page is auto-generated from the GraphQL schema obtained from the source code.
+    #    For any issues and updates, contribute to the corresponding module directly.
     # Can be a Title (for the Nav panel) + file that gets loaded (for the Content panel)
     # - title: Content from file that supports markdown
-    #   file: ./readme.md
+    # - file: spectaql/intro.md
```

Comment:
I thought we were going to use an .md file for the intro. Maybe it's unnecessary if the description is all we need, but it would be nice to see a staging build. Since we're in beta right now for 2.4.6 and have docs deployed to developer-stage.adobe.com for this repo, please use the GitHub Pages environment instead.